packages/compass-preferences-model/package.json

```diff
@@ -60,7 +60,7 @@
     "lodash": "^4.17.21",
     "react": "^17.0.2",
     "yargs-parser": "^21.1.1",
-    "zod": "^3.22.3"
+    "zod": "^3.24.4"
```

Comment:
nit: not sure why we are using it directly instead from `compass-user-data` package, which is already used in this.